README.md

```diff
@@ -316,6 +316,54 @@ The relayer consists of the following components:
   <img src="resources/relayer-diagram.png?raw=true"></img>
 </div>
 
+### API
+
+#### `/relay`
+- Used to manually relay a warp message. The body of the request must contain the following JSON:
+```json
+{
+ "blockchain-id": "<cb58-encoding of blockchain ID>",
+ "message-id": "<Hex encoding of warp message ID>",
+ "block-num": <Integer representation of the block number that the message was sent in>
```

Comment:
I guess you left off the quotes around the value because it's an integer instead of a string, but the markdown render is in some weird highlight like its an error or something, pretty distracting.